model/atmosphere/dycore/tests/dycore_stencil_tests/test_compute_first_vertical_derivative.py

```diff
@@ -22,6 +22,13 @@
 from icon4py.model.testing.helpers import StencilTest
 
 
+def compute_first_vertical_derivative_numpy(
+    z_exner_ic: np.ndarray, inv_ddqz_z_full: np.ndarray
+) -> np.ndarray:
+    z_dexner_dz_c_1 = (z_exner_ic[:, :-1] - z_exner_ic[:, 1:]) * inv_ddqz_z_full
+    return z_dexner_dz_c_1
+
+
 class TestComputeFirstVerticalDerivative(StencilTest):
     PROGRAM = compute_first_vertical_derivative
```

Comment:
See my comment above, I think this test is the only place where the function `compute_first_vertical_derivative` is used, since you inlined the code in `compute_first_vertical_derivative.py`